source/app/util/custom_resource.py

```diff
@@ -61,6 +61,11 @@ def request_id(self):
     def resource_properties(self):
         return self.event.get("ResourceProperties", {})
 
+    # Returns the previous resource properties of the custom resource, these are used to customize the updates
+    @property
+    def old_resource_properties(self):
+        return self.event.get("OldResourceProperties", {})
```

Comment:
What custom resource is involved in this process? I thought it was an event from eventbridge